cbsurge/components/builtenv/electricity/__init__.py

```diff
@@ -0,0 +1,132 @@
+import os
+from abc import ABC
+import geopandas as gpd
+import numpy as np
+
+from cbsurge.core.variable import Variable
+from cbsurge.project import Project
+from cbsurge.util.http_get_json import http_get_json
+import httpx
+import logging
+
+
+
+from cbsurge.core.component import Component
+
+
+
+logger = logging.getLogger(__name__)
+
+class ElectricityGrid(Component, ABC):
+
+    dataset_url = 'https://geohub.data.undp.org/api/datasets/310aadaa61ea23811e6ecd75905aaf29'
```

Comment:
I think we need to register variables at config.json first.

geohub endpoint (https://geohub.data.undp.org) is registered as `geohub_endpoint` property in config.json, so we should reuse this endpoint property.

We can define geohub dataset url like `geohub:{dataset id}`, then resolve this as normal URL.

variables can be defined like below.

```json
{
  "variables": {
     "electricity_grid": {
          "electricity_grid_length": {
                "title": "Total length of electricity grid",
                "source": "geohub:310aadaa61ea23811e6ecd75905aaf29",
                "operator": "sum"
            },
        
      }
  }
}
```